README.md

```diff
@@ -82,20 +82,20 @@ By using the `/examples/nginx.conf`, you could start the Nginx with LUA module o
 Then you could
 1. See the `register logs` happens on the console log.
 ```
-2020/02/27 09:22:06 [debug] 20676#11799235: *4 [lua] content_by_lua(nginx.conf:118):4: Service register request = {"services":[{"type":"normal","serviceName":"User Service Name"}]}
-2020/02/27 09:22:06 [debug] 20676#11799235: *2 [lua] client.lua:99: registerService(): Service register response = [{"key":"User Service Name","value":1}]
+2020/04/04 15:15:37 [debug] 12089#1446111: *4 [lua] content_by_lua(nginx.conf:175):4: Instance report request = {"service":"User Service Name","serviceInstance":"User Service Instance Name","properties":[{"language":"Lua"}]}
```

Comment:
This isn't register logs, please change the title, this is instance properties update logs.